ERCS/erc-7751.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: 7751
+title: Wrapping of bubbled up reverts
+description: Handling bubbled up reverts using custom errors with additional context
+author: Daniel Gretzke (@gretzke), Sara Reynolds (@snreynolds), Alice Henshaw (@hensha256), Marko Veniger <marko.veniger@tenderly.co>, Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/wrapping-of-bubbled-up-reverts/20740
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7751-wrapping-of-bubbled-up-reverts/20740
```
Added ERC number to topic title in Eth Magicians.

Reply:
done